.pre-commit-config.yaml

```diff
@@ -75,7 +75,9 @@ repos:
         entry: pydocstyle
         language: system
         types: [python]
-  - repo: https://github.com/kynan/nbstripout
-    rev: 0.6.0
-    hooks:
-      - id: nbstripout
+      - id: jupyter-nb-clear-output
```

Comment:
i grabbed this one in a different repo because it didn't require any other repo or tool. and then i just copy/pasted it here. i have 0 feelings either way.